verto/processors/PanelBlockProcessor.py

```diff
@@ -0,0 +1,42 @@
+from verto.processors.GenericContainerBlockProcessor import GenericContainerBlockProcessor
+from verto.errors.ArgumentMissingError import ArgumentMissingError
+import re
+
+
+class PanelBlockProcessor(GenericContainerBlockProcessor):
+    def __init__(self, *args, **kwargs):
+        '''
+        Args:
+            ext: An instance of the Verto Extension.
+        '''
+        self.processor = 'panel'
+        super().__init__(self.processor, *args, **kwargs)
+
+    def custom_parsing(self, content_blocks, argument_values):
+        extra_args = {}
+        blocks = []
+
+        argument = 'title'
+        title_r = re.compile(r'(^|\n)# ((\w| )*)(?P<args>)')
+        title = title_r.search(content_blocks[0])
+        if title:
+            extra_args[argument] = title.groups()[1]
+        else:
+            raise ArgumentMissingError(self.processor, argument, '{} is a required argument.'.format(argument))
+
+        argument = 'subtitle'
+        if argument_values.get(argument) == 'true':
+            subtitle_r = re.compile(r'(^|\n)## ((\w| )*)(?P<args>)')
+            subtitle = subtitle_r.search(content_blocks[1])
+            if subtitle:
+                extra_args[argument] = subtitle.groups()[1]
+                blocks = content_blocks[2:]
+            else:
+                raise ArgumentMissingError(self.processor, argument, '{} is set to "true" but not supplied.'.format(argument))  # noqa: E501
```

Comment:
Do we want this to still be ArgumentMissingError when it's not technically an argument? Or should we make a new exception type? I'm not too worried either way but worth mentioning

Reply:
yea this felt a bit wrong to me, I decided to think of it like a required parameter with a default value of `True`, hence spazzing like any other parameter if it's not given. Maybe @JackMorganNZ can give us a decision? I think it could go either way.

Reply:
I think it would be clearer as a new exception (possibly `ArgumentInvalidError` or `ArgumentValueError`), especially as Verto is confusing enough as it is!

Reply:
I think it would be clearer as a new exception (possibly `ArgumentInvalidError` or `ArgumentValueError`), especially as Verto is confusing enough as it is!